src/util/requester.ts

```diff
@@ -237,17 +237,26 @@ export class Requester {
         .labels(dataProviderMetricsLabel(response.status, config.method))
         .inc()
     } catch (e) {
+      const err = e as AxiosError
+      logger.info({
+        msg: 'Request failed',
+        response: {
+          statusCode: err.response?.status,
+          data: err.response?.data,
+          text: err.response?.statusText,
+        },
+      })
```

Comment:
Was this intentionally marked as `info` since this is a data provider error?

Reply:
Correct